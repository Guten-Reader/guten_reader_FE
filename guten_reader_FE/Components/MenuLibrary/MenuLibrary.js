import React from 'react';
import { View, Button, StyleSheet } from 'react-native';
import { Toolbar } from 'react-native-material-ui';

export default function MenuLibrary(props) {
    return (
      <View style={styles.toolbar}>
      <Toolbar
        searchable={{
        autoFocus: true,
        placeholder: 'Search',
      }}
    />
    </View>
    )
}

const styles = StyleSheet.create({
  toolbar: {
    alignItems: 'center',
    color:'#53E69B',
    position: 'absolute',
    bottom: 0,
    right: 0,
    left: 0
  }
})
