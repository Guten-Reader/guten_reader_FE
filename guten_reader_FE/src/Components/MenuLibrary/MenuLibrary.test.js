import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import MenuLibrary from './MenuLibrary';

describe('MenuLibrary renders correctly with Jest', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<MenuLibrary />);
    expect(wrapper).toMatchSnapshot();
  })
})