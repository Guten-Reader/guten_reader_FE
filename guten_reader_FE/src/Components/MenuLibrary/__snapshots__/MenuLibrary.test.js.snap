// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuLibrary renders correctly with Jest renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
